bokeh/sampledata/us_counties.py

```diff
@@ -20,7 +21,15 @@
 nan = float('NaN')
 
 data = {}
-with open(join(data_dir, 'US_Counties.csv')) as f:
+
+# csv differs in Python 2.x and Python 3.x. Open the file differently in each.
+filename = join(data_dir, 'US_Counties.csv')
+if sys.version_info[0] < 3:
+    f = open(filename, 'rb')
+else:
+    f = open(filename, 'r', newline='', encoding='utf8')
+
```

Comment:
Perhaps using `io.open()` would remove the need for this `if-else` statement? It supports `encoding` on both Py 2 and 3.